ui/pages/confirmations/utils/confirm.ts

```diff
@@ -2,6 +2,11 @@ import { ApprovalRequest } from '@metamask/approval-controller';
 import { ApprovalType } from '@metamask/controller-utils';
 import { Json } from '@metamask/utils';
 
+export const ConfirmationRedesignTypes: (ApprovalType | string)[] = [
+  ApprovalType.PersonalSign,
+  ApprovalType.EthSignTypedData,
+];
```

Comment:
From a type perspective, we might want to keep this the same as it is now. In fact, when it comes to arrays like this I'm curious whether this ought to be defined using `as const` rather than using a type annotation here, so that TypeScript has more information on what the allowed types are when this array is used (for instance, if we were to loop over this list we might want to know that the type of each element is `ApprovalType.PersonalSign | ApprovalType.EthSignTypedData` rather than just `string`). So I would suggest:

``` suggestion
export const ConfirmationRedesignTypes = [
  ApprovalType.PersonalSign,
  ApprovalType.EthSignTypedData,
] as const;